test/jdk/java/util/zip/DeInflate.java

```diff
@@ -118,21 +118,33 @@ static void checkByteBufferReadonly(Deflater def, Inflater inf,
         } catch (ReadOnlyBufferException robe) {}
     }
 
-    static void check(Deflater def, byte[] in, int len,
-                      byte[] out1, byte[] out2, boolean nowrap)
+    static void check(Deflater def, byte[] in, int len, boolean nowrap)
         throws Throwable
     {
-        Arrays.fill(out1, (byte)0);
-        Arrays.fill(out2, (byte)0);
-
+        byte[] tempBuffer = new byte[len];
         def.setInput(in, 0, len);
         def.finish();
-        int m = def.deflate(out1);
+        int m = 0;
 
+        ByteArrayOutputStream baos = new ByteArrayOutputStream(len);
```

Comment:
Is there a reason a try-with-resources is not being used?

Reply:
Thanks for pointing it out. Please take a look at the latest modifications.

Reply:
The latest code changes look OK to me. I've added a comment about the test method comment.

Reply:
Tests continue to pass on mach5 with these latest changes.